src/backend/app.js

```diff
@@ -21,6 +23,72 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+app.get("/my-route", (req, res) => {
+  res.send("Hi friend");
+});
+
+//Respond with all meals in the future (relative to the when datetime)
+
+app.get('/future-meals', async(req, res) => {
```

Comment:
These routes should be created in api/meals.js file since there is a router created for meals api.
We will touch this part again during the class